en/docs/observability-and-monitoring/overview-of-observability.md

```diff
@@ -0,0 +1,51 @@
+# Overview of Observability and Monitoring
+
+Observability is a measure of how well the internal states of a system can be understood from its external outputs.
+
+In WSO2 Integrator: BI, observability is a core feature that helps monitor, debug, and optimize integration services. It focuses on the following three key pillars:
+
+<b>Metrics</b> – Numeric data collected and aggregated over time to monitor system performance.
+
+<b>Tracing</b> – Tracking the flow of requests or messages through various services and components, from entry to exit.
+
+<b>Logging</b> – Text-based records of application behavior, annotated with timestamps and contextual information.
+
+Observability platforms allow developers and operators to gain insight into system behavior, troubleshoot issues, and ensure reliability in production deployments.
+
+## Observability in WSO2 Integrator: BI
+
+WSO2 Integrator: BI provides built-in support for observability across its runtime. Integration services, APIs, and connectors emit rich telemetry data that can be exported to standard monitoring tools.
+
+## Available Observability Options
+
+WSO2 Integrator: BI supports multiple options for observing and monitoring deployed integrations. Depending on the deployment environment and the level of visibility required, you can choose from the following observability solutions.
+
+### 1. **WSO2 Integrator: ICP (Integration Control Plane)**
+
+The [WSO2 Integrator: ICP](https://wso2.com/integrator/icp/) provides centralized monitoring and management of runtime artifacts across a deployment. It offers a graphical interface to view deployed integration artifacts and their relationships, enabling teams to,
+
+* Track the status and availability of deployed services.
+* View runtime metadata and node-level details.
+* Manage and visualize the health of integration components across environments.
+
+This is ideal for teams that manage integrations across hybrid or distributed environments and require control-plane-level visibility.
```

Comment:
Add link for observability-and-monitoring/monitoring-with-wso2-integrator-icp/ to refer.